management/actions_test.go

```diff
@@ -12,6 +12,8 @@ import (
 )
 
 func TestActionManager_Create(t *testing.T) {
+	setupVCR(t)
```

Comment:
Setting up the VCR like this allows us to have a separate recording for each test, giving us higher flexibility in terms of needing to change things between scenarios.